contrib/docs/tutorials/enable-ssl-self-signed.md

```diff
@@ -0,0 +1,47 @@
+# Enabling SSL for Local Testing
+
+If you need to use an `https:` URL for local testing (i.e. if an OAuth provider requires a "secure" callback URL), you can use a self-signed certificate by following these steps.
+
+## Backend
+
+1. Generate a self-signed certificate and key for localhost and configure your system to trust it. The application [`mkcert`](https://github.com/FiloSottile/mkcert) is a helpful tool to accomplish this.
+1. Update `backend.baseUrl` in app-config.local.yaml to use an `https:` address.
+1. Add the certificate and key to `backend.https.certificate.cert` and `backend.https.certificate.cert`, respectively.
+   ```yaml
+   backend:
+     baseUrl: https://localhost:7007
+     https:
+       certificate:
+         # You may copy the contents of the file...
+         cert: |
+           -----BEGIN CERTIFICATE-----
+           MIIDCTCCAfGgAwIBAgIUZ9VhZckcy690L
+           ...
+           -----END CERTIFICATE-----
+         # ... or use a path
+         key:
+           $file: ./certs/localhost-key.pem
+   ```
+1. Start the backend with `NODE_EXTRA_CA_CERTS=/absolute/path/to/cert.pem yarn start-backend`
```

Comment:
is this needed if you use mkcert as it installs the certificate in the trust store? Would you be able to verify if it works without it? 🙏